src/redux/books/books.js

```diff
@@ -0,0 +1,25 @@
+// Actions
+const ADD_BOOK = 'ADD_BOOK';
+const REMOVE_BOOK = 'REMOVE_BOOK';
+
+// Action Creators
+export const addBook = (book) => ({ type: ADD_BOOK, payload: book });
+export const removeBook = (id) => ({ type: REMOVE_BOOK, payload: { id } });
+
+// Reducer
+const books = (state = [], action) => {
```

Comment:
- [ ] Instead initializing the state here directly, kindly assign the empty array here to a variable like `initialState` and then assign state here to `initialState` like so: `const books = (state = initialState, action) => {`.

  That is better because in some applications we may need to set an initial state with actual values. Doing so here, in the parameter, would be messy.

- [ ] **[OPTIONAL]** I think it would be more descriptive to rename this to `reducer` instead of `books`